tests/test_configs.py

```diff
@@ -0,0 +1,25 @@
+import kernelci.configs
+
+def test_build_configs_parsing():
+    """ Verify build-configs.yaml """
+    configs = kernelci.configs.builds_from_yaml("build-configs.yaml")
+    assert len(configs) == 4
```

Comment:
Might be worth checking that the required things are in the returned dict (`trees`, `fragments`...).

Reply:
Good suggestion. Added